recipe/build-nocgo-platform.sh

```diff
@@ -13,10 +13,5 @@ for F in activate deactivate; do
     -e "s|@GOARCH@|${conda_goarch}|g" "${F}-go-platform.sh"
 
   # Copy the rendered [de]activate scripts to $PREFIX/etc/conda/[de]activate.d
-  cp -v "${F}-go-platform.sh" "${PREFIX}/etc/conda/${F}.d/${F}-${PKG_NAME}.sh"
+  cp -v "${F}-go-platform.sh" "${PREFIX}/etc/conda/${F}.d/${F}-z60-${PKG_NAME}.sh"
```

Comment:
I am using `z60` to make our activate script files be loaded after the gcc, fortran and g++ activation scripts.